content/documentation/terminology/index.adoc

```diff
@@ -0,0 +1,167 @@
+---
+title: "Terminology"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a runtime/program that runs in the environment.
+
+== App Name
+
+It's the name of the <<Application>> deployed in your environment.
+
+== App Label
+
+This represents the app label of an object.
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+
+=== Use Case
+
+- Create a Deployment to rollout a ReplicaSet. The ReplicaSet creates Pods in the background. Check the status of the rollout to see if it succeeds or not.
```

Comment:
Not sure if "Use Case" and the definition below is clear enough; I understand that those are "examples" of use cases, but not sure what is best way to describe them.